src/browser-connection/local.js

```diff
@@ -29,14 +29,17 @@ export default class LocalBrowserConnection extends BrowserConnection {
 
 
     close () {
+        //NOTE: for the local connection we should close a browser before closing
+        // the connection, so we close the connection in the getStatus function
```

Comment:
When using local connections, we should close the browser before closing the connection. For this, we close connections in the getStatus function.